.github/workflows/release.yml

```diff
@@ -17,37 +17,37 @@ jobs:
     strategy:
       matrix:
         build:
-          - os: ubuntu-20.04
+          - os: ${{ fromJson(github.repository_owner == 'subspace' && github.repository == 'subspace/subspace-cli' && '["self-hosted", "ubuntu-22.04-x86-64", "Linux"]' || '["ubuntu-22.04"]') }}
```

Comment:
This and below is wrong too. It must use Ubuntu 20.04, not 22.04. Using 22.04 means our executables will fail to work on Ubuntu 20.04 due to old Glibc. There is a reason behind every line and version in this file, don't change it arbitrarily.

Reply:
:cry:  will setup a new linux runner with the older version, didn't know the 20.04 is a hard requirement.

Reply:
Well, if we change the built to happen in a container with Ubuntu 20.04 userspace we can do it on existing workers, but we can't do it on the host I'm afraid (maybe there is a way to limit Glibc version for builds, I don't know).

This doesn't matter for tests, but for releases we need 20.04.